### PR TITLE
Replace deprecated key vault libraries

### PR DESCRIPTION
**Pull request by 12step**
Hopefully changes to startup and appsettings show up in the PR

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -354,3 +354,7 @@ healthchecksdb
 
 /src/Server/BlazorBoilerplate.Server/Modules
 /src/Server/BlazorBoilerplate.Server/Keys
+/src/Server/BlazorBoilerplate.Server/Properties/PublishProfiles
+/src/Server/BlazorBoilerplate.Server/Properties/ServiceDependencies/ClaimCentre - Web Deploy/profile.arm.json
+/src/Server/BlazorBoilerplate.Server/appsettings.Development.json
+/src/Server/BlazorBoilerplate.Server/appsettings.Production.json
```
Comment (GioviQ) [inline .gitignore:360]: appsettings ... should not be ignored

File: README.md [hunk 2]
```diff
@@ -1,7 +1,6 @@
 # <img src="https://github.com/enkodellc/blazorboilerplate/blob/master/docs/images/logo-title.png" alt="Blazor Boilerplate" style="max-wdith:100%"/>
 
-Blazor is a web framework designed to run in the browser on a WebAssembly-based .NET runtime. Blazor Boilerplate aka Blazor Starter Template is a SPA admin template that is able to run both WebAssembly (Core-Hosted) and Server-Side Blazor with a .NET 5.0 Server. Just set the desidered runtime in admin section.
-The UI for this application is by Material Design provided mostly by MatBlazor.
+Blazor is a web framework designed to run in the browser on a WebAssembly-based .NET runtime. Blazor Boilerplate aka Blazor Starter Template is a SPA admin template that is able to run both WebAssembly (Core-Hosted) and Server-Side Blazor with a .NET 5.0 Server. Default mode for BB is Server Side. To switch to Webassembly log in as Admin and go to settings. [Read more here](https://blazor-boilerplate.readthedocs.io/en/latest/features/dual_mode_blazor.html)
```
Comment (GioviQ) [inline README.md:3]: It seems you did not merge the last commits

File: src/Server/BlazorBoilerplate.Server/Properties/launchSettings.json [hunk 3]
```diff
@@ -3,7 +3,7 @@
     "windowsAuthentication": false,
     "anonymousAuthentication": true,
     "iisExpress": {
-      "applicationUrl": "http://localhost:53414/",
+      "applicationUrl": "http://localhost:53555/",
```
Comment (GioviQ) [inline src/Server/BlazorBoilerplate.Server/Properties/launchSettings.json:6]: Why ?

File: src/Server/BlazorBoilerplate.Server/Properties/launchSettings.json [hunk 4]
```diff
@@ -23,7 +23,7 @@
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
       },
-      "applicationUrl": "http://localhost:53416/"
+      "applicationUrl": "http://localhost:53556/"
```
Comment (GioviQ) [inline src/Server/BlazorBoilerplate.Server/Properties/launchSettings.json:26]: Why?

File: src/Shared/Modules/BlazorBoilerplate.Theme.Material.Admin/Pages/Admin/IdentityResources.razor [hunk 5]
```diff
@@ -12,15 +12,15 @@
 <h1>@L["IdentityResources"]</h1>
 <p>Identity Resources Management.</p>
 
-@if (identiyResources == null)
+@if (identityResources == null)
```
Comment (GioviQ) [inline src/Shared/Modules/BlazorBoilerplate.Theme.Material.Admin/Pages/Admin/IdentityResources.razor:15]: This file should not be included in PR.

File: src/Server/BlazorBoilerplate.Server/Startup.cs [hunk 6]
```diff
@@ -39,8 +46,8 @@
 using Microsoft.AspNetCore.Identity;
 using Microsoft.AspNetCore.Localization;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.Azure.KeyVault;
-using Microsoft.Azure.Services.AppAuthentication;
+//using Microsoft.Azure.KeyVault; DEPRECATED, using Azure.Security.KeyVault replacement libraries https://www.nuget.org/packages/Microsoft.Azure.KeyVault
```
Comment (GioviQ) [inline src/Server/BlazorBoilerplate.Server/Startup.cs:49]: To remove completly if it is not used anymore

File: src/Server/BlazorBoilerplate.Server/appsettings.json [hunk 7]
```diff
@@ -1,20 +1,27 @@
...
+    //   Initial Catalog=blazor-boilerplate-demo-db;User Id=<adminusername>; Password=<adminuserpassword>; Trusted_Connection=False; Encrypt=True; 
+    //   MultipleActiveResultSets=True;
+    "DefaultConnection": "Data Source=(localdb)\\MSSQLLocalDB;Initial Catalog=BlazorBoilerplate;Trusted_Connection=True;MultipleActiveResultSets=True"
   },
 
   "HostingOnAzure": {
-    "RunsOnAzure": false,
-    "RunningAsAppService": true,
-    "RunningAsDocker": false, // not implementet yet
+    "RunsOnAzure": false, // true for Azure hosting
```
Comment (GioviQ) [inline src/Server/BlazorBoilerplate.Server/appsettings.json:11]: Please remove obvious comments

**Review by GioviQ — COMMENTED** [review]
Please merge first last commits and check my comments.

File: src/Shared/Modules/BlazorBoilerplate.Theme.Material.Admin/Pages/Admin/IdentityResources.razor (hunk 5 above)
Comment (12step) [reply src/Shared/Modules/BlazorBoilerplate.Theme.Material.Admin/Pages/Admin/IdentityResources.razor:15]: Agree that was a separate fix I did to rename misspelled idenity to identity. I will send in separate PR.

**Pull request closed without merging**